Win/Mineways.cpp

```diff
@@ -1478,6 +1499,24 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
                 }
             }
             break;
+        case IDM_FILE_PUBLISHSKFB:
+            // Force it to be an rendering export: Relative obj
+            gPrintModel = 0;
+            gExportViewData.fileType = FILE_TYPE_WAVEFRONT_REL_OBJ;
```

Comment:
it seems that the convention is to not use space before and after assignment; be consistent with existing codebase please